Cargo.toml

```diff
@@ -10,11 +10,11 @@ name = "stderrlog"
 readme = "README.md"
 changelog = "CHANGELOG.md"
 repository = "https://github.com/cardoe/stderrlog-rs"
-version = "0.2.4-pre"
+version = "0.3.0"
```

Comment:
I use `cargo-release` for this so you don't need to mess with it. This will actually become 0.4.0